src/_base/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# CHANGELOG
+
+## 1.5.0 - 20230126
+
+### Added attributes
+
+* `attribute('console.healthcheck.enabled')` - Defaults to `true` only for local builds
+* `attribute('console.healthcheck.action.steps')` - Steps necessary to create a measurable condition to test later
+* `attribute('console.healthcheck.test.script')` - The path to the script that will host the health checks steps
+* `attribute('console.healthcheck.test.command')` - The command in array format that will run the healthcheck - Defaults to the test script path
+* `attribute('console.healthcheck.test.steps')` - Steps necessary to test the measurable condition to determine the health of the container
+* `attribute('console.healthcheck.test.start_period')` - Defaults to 5s
+* `attribute('console.healthcheck.test.interval')` - Defaults to 5s
+* `attribute('console.healthcheck.test.timeout')` - Defaults to 3s
+* `attribute('console.healthcheck.test.retries')` - Defaults to 10
```

Comment:
service.console.healthcheck for standard service use

Reply:
I originally used that 
but give that `entrypoint` and `healthcheck` are Docker components at the same level
and that the `default` (empty) values for the `entrypoint` where set at the layer higher then `services` I thought it would be consistent to put the `healthcheck` at the same level as `entrypoint`

is that not appropriate?

Reply:
hmm I see `console...` already in use, so can see why it is as it is